src/main/java/uk/gov/hmcts/reform/civil/service/docmosis/utils/ClaimantResponseUtils.java

```diff
@@ -92,7 +92,7 @@ private static long getNumberOfInstallmentsAfterFirst(BigDecimal totalAmount, Bi
         return totalAmount.divide(paymentAmount, 0, RoundingMode.CEILING).longValue() - 1;
     }
 
-    private static BigDecimal getDefendantAdmittedAmount(CaseData caseData) {
+    public static BigDecimal getDefendantAdmittedAmount(CaseData caseData) {
```

Comment:
This utils should be moved from /civil/service/docmosis/utils to civil/utils because it is not a docmosis specific utility, and it is usable in the whole project

Reply:
As we agreed, it will be refactor in your branch